pkg/util/containers/v2/metrics/docker/collector_linux.go

```diff
@@ -28,7 +28,6 @@ func convertContainerStats(stats *types.Stats) *provider.ContainerStats {
 		CPU:       convertCPUStats(&stats.CPUStats),
 		Memory:    convertMemoryStats(&stats.MemoryStats),
 		IO:        convertIOStats(&stats.BlkioStats),
-		PID:       convertPIDStats(&stats.PidsStats),
```

Comment:
Just for my understanding, why are we deleting this line? Don't we need the `ThreadCount` and `ThreadLimit` anymore?

Reply:
Indeed, good catch